.changeset/a11y-accordion.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-dom': patch
```

Comment:
Although technically we are updating react-magma-dom, this is a documentation only change, so let's make this `'react-magma-docs': patch`. The only reason I'm recommending this is because, if it's labeled under dom, it will become part of our user facing changelog when there's no actual change to the component. Otherwise, the change looks great!